peps/converters.py

```diff
@@ -7,11 +7,16 @@
 from django.core.exceptions import ImproperlyConfigured
 from django.core.files import File
 
+from pygments import highlight
+from pygments.lexers import PythonLexer
+from pygments.formatters import HtmlFormatter
+
 from pages.models import Page, Image
 
 PEP_TEMPLATE = 'pages/pep-page.html'
 pep_url = lambda num: 'dev/peps/pep-{}/'.format(num)
 
+PURE_PYTHON_PEPS = [483, 484, 526]
```

Comment:
This is probably not important to have a constant. I'd say either inline it or add a comment to explain why we need it.